Simplenote/Classes/UIView+Animations.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+import UIKit
+
+
+// MARK: - UIView's Animation Methods
+//
+extension UIView {
+
+    /// Indicates if the receiver has the horizontal compact trait
```

Comment:
Maybe update the comment?  It looks like it was copied from `UIView+Simplenote.swift`.

Reply:
Nice catch, fixed, thank you!!